btree/btree.c

```diff
@@ -11991,6 +11995,10 @@ static void ut_lru_test(void)
 	M0_ASSERT(ndlist_tlist_length(&btree_lru_nds) > 0);
 
 	mem_freed      = m0_btree_lrulist_purge(mem_increased/2);
+	/** Adding sleep here to let the memory get updated after freeing up of
+	 *  memory in lrulist_purge.
+	 */
+	sleep(10);
```

Comment:
Why is this sleep needed ?

Reply:
Added it as a precaution to make sure that memory gets updated after calling lrulist_purge().

Reply:
No this should not be done because on a running system we will not have any sleep's so if there are issues to be found then they need to be found now without sleep.

Reply:
Ok, I will remove it.